src/strategies/ocean-dao-brightid/index.ts

```diff
@@ -27,16 +27,20 @@ export async function strategy(
     options.strategies.some((x) => !ALLOWED_STRATEGIES.includes(x.name))
   )
     return [];
+
+  const brightIdNetwork = options.brightIdNetwork || network;
+  const brightIdProvider = getProvider(brightIdNetwork);
+  const brightIdSnapshotBlock = options.brightIdSnapshotBlock || snapshot;
```

Comment:
Even `options.brightIdSnapshotBlock` has to be changed in space settings every time, now I feel it is better to make something like `multichain` strategy, where you can get block number of brightID based on snapshot block timestamp

https://github.com/snapshot-labs/snapshot-strategies/blob/master/src/strategies/multichain/index.ts#L38-L61

Reply:
Thanks, I've updated the strategy.